synapse/lib/stormtypes.py

```diff
@@ -234,6 +235,20 @@ def addLibFuncs(self):
         row = [toprim(a) for a in args]
         await self.runt.snap.fire('csv:row', row=row, table=table)
 
+class LibBase64(Lib):
+
+    def addLibFuncs(self):
+        self.locls.update({
+            'encode': self._encode,
```

Comment:
why are these not an extension of the Bytes storm primitive?

Reply:
We discussed this a bit.  I'm open to revisiting.  Lets goback about it today.